style.css

```diff
@@ -0,0 +1,1567 @@
+  box-shadow: inset 0 -1px 0 rgba(255, 68, 34, 0.25);
+  color: #f42;
+}
+
+.story-byline {
+  color: #aaa;
+  font-size: 12px;
+  line-height: 18px;
+  padding-top: 4px;
+}
+
+.story-byline::after {
+  clear: both;
+  content: "";
+  display: block;
+}
+
+.story-byline li {
+  display: block;
+  float: left;
+}
+
+.story-byline li::after {
+  content: "\B7";
+  display: block;
+  float: right;
+  margin: 0 4px;
+}
+
+.story-byline li:last-child::after {
+  display: none;
+}
+
+.story-byline a {
+  color: #aaa;
+}
+
+.story-byline a:hover {
+  color: #555;
+}
+
+@media (max-width: 767px) {
+  .story--smallMobile {
+    min-height: 72px;
+    padding-left: 88px;
+    position: relative;
+  }
+
+  .story--smallMobile .story-image {
+    left: 0;
+    padding-top: 75px;
+    position: absolute;
+    top: 0;
+    width: 75px;
+  }
+
+  .story--smallMobile .story-title {
+    font-size: 14px !important;
+    font-weight: 700;
+    line-height: 18px !important;
+  }
+
+  .story--smallMobile .story-byline {
+    line-height: 18px !important;
+  }
+}
+
+@media (min-width: 1024px) {
+  .story .story-title {
+    font-size: 16px;
+    line-height: 20px;
+    margin-bottom: 0;
+  }
+
+  .story .story-byline {
+    font-size: 13px;
+    line-height: 20px;
+  }
+}
+
+a,
+div,
+h2,
+h4,
+header,
+li,
+section,
+span,
+ul {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font: inherit;
+  font-size: 100%;
+  vertical-align: baseline;
+}
+
+ul {
+  list-style: none;
+}
+
+header,
+section {
+  display: block;
+}
+
+*,
+::after,
+::before {
+  box-sizing: border-box;
+  -webkit-font-smoothing: antialiased;
+  outline: 0;
+  text-rendering: optimizeLegibility;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  color: #000;
+}
+
+h2 {
+  font-family: Avalon, sans-serif;
+  font-size: 28px;
+  letter-spacing: -0.015em;
+}
+
+h2 {
+  font-weight: 700;
+  line-height: 1.1;
+}
+
+h4 {
+  font-weight: 700;
+  line-height: 1.25;
+}
+
+h4 {
+  font-size: 14px;
+}
+
+@media (min-width: 768px) {
+  h2 {
+    font-size: 28px;
+  }
+}
+
+.u-row {
+  display: -webkit-flex;
+  display: flex;
+  -webkit-flex: 0 1 auto;
+  flex: 0 1 auto;
+  -webkit-flex-flow: row wrap;
+  flex-flow: row wrap;
+  margin: -8px;
+}
+
+[class*=" u-col-"],
+[class^=u-col-] {
+  -webkit-flex: 0 0 auto;
+  flex: 0 0 auto;
+  padding: 8px;
+  position: relative;
+}
+
+.u-col-xs-12 {
+  -webkit-flex-basis: 100%;
+  flex-basis: 100%;
+  max-width: 100%;
+}
+
+@media (min-width: 768px) {
+  .u-row {
+    margin: -10px;
+  }
+
+  [class*=u-col-] {
+    padding: 10px;
+  }
+
+  .u-col-m-4 {
+    -webkit-flex-basis: 33.3333333333%;
+    flex-basis: 33.3333333333%;
+    max-width: 33.3333333333%;
+  }
+}
+
+@media (min-width: 1024px) {
+  .u-row {
+    margin: -12px;
+  }
+
+  [class*=u-col-] {
+    padding: 12px;
+  }
+
+  .u-col-l-3 {
+    -webkit-flex-basis: 25%;
+    flex-basis: 25%;
+    max-width: 25%;
+  }
+}
+
+.deals {
+  margin-bottom: 2em;
+  margin-left: 380px;
+  margin-right: 380px;
```

Comment:
Why do we have this big margin?

Consider reducing it, currently, its content is rendered with a very small space.

| Source | Clone |
|:----:| :---: |
|![lastest-deals-source](https://user-images.githubusercontent.com/36057474/65874592-ea219f80-e37d-11e9-84f8-aab61b6c8947.png)  | ![lastest-deals-clone](https://user-images.githubusercontent.com/36057474/65874622-f148ad80-e37d-11e9-89d0-6747773f6151.png) |


### Also. remember to make it responsive.